app/src/main/res/layout/fragment_movies_list.xml

```diff
@@ -4,29 +4,20 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/dark_grey">
-
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="@dimen/activity_standard_margin"
-        android:layout_marginTop="@dimen/content_title_top_margin"
-        android:drawableLeft="@drawable/ic_target"
-        android:drawablePadding="@dimen/activity_medium_margin"
-        android:text="@string/movies_list_title"
-        android:textColor="@color/white"
-        android:textSize="@dimen/text_medium_size"
-        android:textStyle="bold" />
+    android:background="@color/dark_grey"
+    android:fitsSystemWindows="true">
 
     <androidx.recyclerview.widget.RecyclerView
         android:id="@+id/movie_list"
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="match_parent"
         android:layout_marginStart="@dimen/activity_standard_margin"
-        android:layout_marginTop="@dimen/content_top_margin"
         android:layout_marginEnd="@dimen/activity_standard_margin"
         android:layout_marginBottom="@dimen/activity_standard_margin"
+        android:overScrollMode="never"
+        android:nestedScrollingEnabled="false"
```

Comment:
I was playing with CollapsingToolbar. Though couldn't make it work (collapse on scrolling) with RecyclerView :(

Reply:
https://github.com/SmartToolFactory/Toolbar-Samples